app/uk/gov/hmrc/agentclientrelationships/connectors/HipConnector.scala

```diff
@@ -147,15 +112,15 @@ class HIPConnector @Inject() (
 
     implicit val reads: Reads[ActiveRelationship] = ActiveRelationship.hipReads
 
-    getRelationship(s"GetActiveClientRelationships", url)
+    getWithHipHeaders(s"GetActiveClientRelationships", url)
       .map {
         case Right(response) =>
           (response.json \ "relationshipDisplayResponse").as[Seq[ActiveRelationship]].find(isActive)
         case Left(errorResponse) =>
           errorResponse.statusCode match {
-            case Status.BAD_REQUEST | Status.NOT_FOUND                                         => None
-            case Status.UNPROCESSABLE_ENTITY if errorResponse.getMessage.contains("suspended") => None
-            case Status.UNPROCESSABLE_ENTITY if errorResponse.getMessage.contains("009")       => None
+            case Status.BAD_REQUEST | Status.NOT_FOUND => None
+            case Status.UNPROCESSABLE_ENTITY if errorResponse.getMessage.contains("suspended") =>
+              None
```

Comment:
did you mean to remove this handler for relationship not found? it was only added this morning to avoid logging these cases as Errors